spec/schema.json

```diff
@@ -16,53 +16,346 @@
+          "in": "path",
+          "required": true,
+          "description": "ID of the Objection",
+          "type": "string"
+        },
+        {
+          "name": "attachmentId",
+          "in": "path",
+          "required": true,
+          "description": "ID of the attachment",
+          "type": "string"
+        }
+      ],
+      "get": {
+        "description": "download this attachment",
+        "produces": [
+          "application/octet-stream"
+        ],
+        "tags": [
+          "Strike Off Objections"
+        ],
+        "responses": {
+          "200": {
+            "description": "the attachment was found",
+            "schema": {
+              "type": "file"
+            }
+          },
+          "404": {
+            "description": "Objection or attachment not found"
           }
         }
       }
     }
   },
   "definitions": {
-    "ObjectionOutput": {
+    "Objection": {
       "type": "object",
       "properties": {
-        "requestId": {
-          "type": "integer",
-          "format": "int64"
+        "id": {
+          "type": "string",
+          "format": "uuid"
+        },
+        "created_on": {
+          "type": "string",
+          "format": "date-time"
+        },
+        "created_by": {
+          "$ref": "#/definitions/CreatedBy"
+        },
+        "reason": {
+          "type": "string"
+        },
+        "attachments": {
+          "type": "array",
+          "items": {
+            "properties": {
+              "link": {
+                "type": "string",
+                "format": "uri"
+              }
+            }
+          }
+        },
+        "status": {
+          "$ref": "#/definitions/Status"
+        }
+      }
+    },
+    "StrikeOffStatus": {
+      "type": "object",
+      "properties": {
+        "isEligible": {
+          "type": "boolean"
+        },
+        "strikeOffDate": {
+          "type": "string"
         }
       }
     },
-    "ObjectionInput": {
+    "ResourceCreated": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "format": "uuid"
+        }
+      }
+    },
+    "Attachment": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "format": "uuid"
+        },
+        "name": {
+          "type": "string"
+        },
+        "content_type": {
+          "type": "string"
+        },
+        "size": {
+          "type": "number"
+        },
+        "links": {
+          "type": "object",
+          "properties": {
+            "download": {
+              "type": "string",
+              "format": "uri"
+            },
+            "self": {
+              "type": "string",
+              "format": "uri"
+            }
+          }
+        }
+      }
+    },
+    "CreatedBy": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string"
+        },
+        "forename": {
```

Comment:
Where / how will we get forename and surname from? I thought that user just signs in with eMail address.

Reply:
we get them from ERIC, I guess we don't technically need them as we can cross reference the user data

Reply:
Fair enough if we have them.

Reply:
i've taken them out, they were just nice to have's